io/plugins/eu.esdihumboldt.hale.io.json/src/eu/esdihumboldt/hale/io/json/GeoJSONInstanceWriter.java

```diff
@@ -22,20 +22,32 @@
 import eu.esdihumboldt.hale.common.core.io.report.IOReport;
 import eu.esdihumboldt.hale.common.core.io.report.IOReporter;
 import eu.esdihumboldt.hale.common.core.io.report.impl.IOMessageImpl;
-import eu.esdihumboldt.hale.common.instance.io.impl.AbstractInstanceWriter;
 
 /**
  * Writes instances as GeoJSON.
  * 
  * @author Kai Schwierczek
  */
-public class GeoJSONInstanceWriter extends AbstractInstanceWriter {
+public class GeoJSONInstanceWriter extends JsonInstanceWriter {
 
 	/**
 	 * Parameter name for the default geometry association.
+	 * 
+	 * @deprecated as of release 4.2.0 because we don't use geometry
+	 *             configuration for geoJson and export the data in WG84 format.
+	 *             As geoJson expects WGS 84 with lon/lat (see
+	 *             https://tools.ietf.org/html/rfc7946)
 	 */
+	@Deprecated
 	public static final String PARAM_GEOMETRY_CONFIG = "geojson.geometry.config";
 
+	/**
+	 * @param geoJson use geoJson feature when exporting to geoJson
+	 */
+	public GeoJSONInstanceWriter() {
+		super(true);
+	}
+
 	@Override
 	public boolean isCancelable() {
```

Comment:
We can remove this method, no additional value here compared to the super class.